tests/snippets/dict.py

```diff
@@ -0,0 +1,16 @@
+def dict_eq(d1, d2):
+    return (all(k in d2 and d1[k] == d2[k] for k in d1)
+            and all(k in d1 and d1[k] == d2[k] for k in d2))
+
+
+assert dict_eq(dict(a=2, b=3), {'a': 2, 'b': 3})
+assert dict_eq(dict({'a': 2, 'b': 3}, b=4), {'a': 2, 'b': 4})
```

Comment:
How is it possible that this file was added? We already have it in the master branch?

Reply:
Yeah, sorry. I rebased and force pushed, so those shouldn't be there anymore.